mercurial.html.markdown

```diff
@@ -0,0 +1,387 @@
+---
+category: tool
+tool: hg
+contributors:
+  - ["Will L. Fife", "http://github.com/sarlalian"]
+filename: LearnHG.txt
+---
+
+Mercurial is a free, distributed source control management tool. It offers
+you the power to efficiently handle projects of any size while using an
+intuitive interface. It is easy to use and hard to break, making it ideal for
+anyone working with versioned files.
+
+## Versioning Concepts
+
+### What is version control?
+
+Version control is a system that keeps track fo changes to a set of file(s)
+and/or directorie(s) over time.
+
+
```

Comment:
Please remove the double newlines here and at other places in the doc. A single empty line would suffice  for this.